### PR TITLE
[#8446] [Multi-Account] Build account settings for individual wallet

### LINKED ISSUE #8446: [Multi-Account] Build account settings for individual wallet
### Description

*Type*: Feature

*Summary*: Build on #8348. Add the [`Account settings` screens](https://www.figma.com/file/XUehMnhyD1FGcWzvGz6SXqvh/Mobile-wallet?node-id=2559%3A33433). 

#### Expected behavior

When viewing a wallet, I can tap on `Account settings` to change the name and color of the wallet. 

image

#### Actual behavior

`Account settings` don't yet exist. Menu item is not tappable.

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #8446

tested on ios simulator, edit account

image

### CODE REVIEW

**Comment by flexsurfer** [comment]
blocked by status-im/status-go#1635

**Comment by andmironov** [comment]
iOS build link does not seem to work, will test on Android tho

**Comment by flexsurfer** [comment]
@andmironov please wait for the latest build,  ios builds should work, @churik @jakubgs ?

**Comment by andmironov** [comment]
Bugs I spotted on Android:
- If I change the name of the account and tap "Apply" the keyboards does not get hidden;
- If I change the name of the account it does not change;
- If I change the color of the account it does not change;
- There is an extra margin before "Account name";
- There's no bottom padding below "Storage: This device" (My Nexus' screen height is exactly the same as the height of this screen);

Here's a screenshot with all the margins and paddings issues I'm talking about

photo_2019-10-16 14 19 43

**Review by andmironov — COMMENTED** [review]
Bugs I spotted on Android:
- If I change the name of the account and tap "Apply" the keyboards does not get hidden;
- If I change the name of the account it does not change;
- If I change the color of the account it does not change;
- There is an extra margin before "Account name";
- There's no bottom padding below "Storage: This device" (My Nexus' screen height is exactly the same as the height of this screen);

Here's a screenshot with all the margins and paddings issues I'm talking about

photo_2019-10-16 14 19 43

**Comment by flexsurfer** [comment]
sorry @andmironov you were reviewing stale build, i've added  bottom padding, but not sure about extra margin before "Account name"; thanks

**Comment by flexsurfer** [comment]
@siphiuel @dmitryn, please review

**Comment by flexsurfer** [comment]
@yenda @siphiuel please review

File: src/status_im/wallet/accounts/core.cljs [hunk 1]
```diff
@@ -87,6 +87,16 @@
             (navigation/navigate-to-cofx :account-added nil)))
 
 (fx/defn save-account
+  {:events [:wallet.accounts/save-account]}
+  [{:keys [db] :as cofx} account {:keys [name color]}]
+  (let [{:keys [accounts]} (:multiaccount db)
+        new-account (cond-> account
```
Comment (vitvly) [inline src/status_im/wallet/accounts/core.cljs:93]: Maybe we can do `(merge account new-account)` here to avoid `name`/`color` duplication.

**Review by vitvly — APPROVED** [review]

**Pull request merged**